package.json

```diff
@@ -57,6 +57,7 @@
     "@types/minimist": "1.2.1",
     "@types/mocha": "8.2.1",
     "@types/node": "14.14.32",
+    "0x": "4.10.2",
```

Comment:
@danielrozenberg Looks like this dependency was inadvertently added.

Reply:
Merging package.json is always such a pain...

Reply:
I know! And this particular dep is weird because different versions of `npm` put it in different places. @alanorozco is being kind enough to take care of it in #33154.